app/views/projects/index.html.erb

```diff
@@ -44,6 +44,7 @@
     <% show_it = (p.status == 'active' || p.status == 'Active' ? true : false) %>
     <% follower_count = p.followers.count %>
     <% commit_count = p.commit_count %>
+    <% last_update = p.last_github_update %>
```

Comment:
We should put in a ticket to take some of this logic and local variable declarations out of this view and put them in helpers

Reply:
To facilitate this, we should have some kind of test of this feature at the cuke level, at least.